app/server/utils/ussd/token_processor.py

```diff
@@ -348,14 +325,11 @@ def get_token_info(transfer_account: TransferAccount):
 
         def check_if_ge_limit(token_info):
             return 'GE Liquid Token' in token_info['limit'].name
-            # return (token_info['exchange_rate'] is not None
-            #         and token_info['limit'] is not None
-            #         and token_info['limit'].transfer_balance_fraction is not None)
 
         def ge_string(t):
-            if t['limit'].transfer_balance_fraction:
-                # TODO: This doesn't seem DRY with respect to 'get default exchange rate'
-                allowed_amount = rounded_dollars(t['limit'].transfer_balance_fraction * t['balance'])
+            if isinstance(t['limit'], BalanceFractionLimit):
+                # Ideally this multiplication wo
+                allowed_amount = rounded_dollars(t['limit'].total_from_balance(t['balance']))
```

Comment:
it annoys me slightly that we have to input the balance here - seems out of pattern with the rest of the limits that always take a credit transfer obj